lib/galaxy/datatypes/hubAssembly.py

```diff
@@ -0,0 +1,59 @@
+"""
+HubAssembly datatype
+"""
+import logging
+import galaxy.version as version
+
+# Support for Galaxy <= 16.01
+if version.VERSION_MAJOR <= "16.01":
+    from galaxy.datatypes.images import Html
+else:
+    from galaxy.datatypes.text import Html
```

Comment:
Technically, importing from `galaxy.datatyes.images` should work in any versions because of [this](https://github.com/galaxyproject/galaxy/pull/1760/files#diff-ee03a0e86725af1de996583c727979ebR301). 
But I preferred to use the text lib, to drop the images one in the future (once we drop support for < 16.04...so in a long time heh).

How do you think we should resolve this?